interop_binaries/src/bin/janus_interop_aggregator.rs

```diff
@@ -161,7 +165,21 @@ async fn make_handler(
             max_upload_batch_size: 100,
             max_upload_batch_write_delay: std::time::Duration::from_millis(100),
             batch_aggregation_shard_count: 32,
-            ..Default::default()
+            global_hpke_configs_refresh_interval: GlobalHpkeKeypairCache::DEFAULT_REFRESH_INTERVAL,
```

Comment:
It bothers me that the interop test stuff needs to be aware of taskprov at all. That doesn't really make sense, since taskprov isn't exercised in the interop test setup and shouldn't be used. If the higher level goal here is to move toward specializing the Janus on `release/0.subscriber-01` to exclusively be a taskprov helper, then I think we could just drop support for the interop testing API altogether.

Reply:
Discussed in slack. We can move forward with this PR. If we have any more intrusive changes to the interop API, we should drop it.